### PR TITLE
initial Axolotl Atlas Implementation 

### PR DESCRIPTION
**Pull request by saimaabdus19**
## Description

**What is this PR**

- [ ] Bug fix
- [x] Addition of a new feature
- [ ] Other

**Why is this PR needed?**

**What does this PR do?**

## References 
Gets us most of the way to #270

## How has this PR been tested?

Local visualisation

## Is this a breaking change?

No

## Does this PR require an update to the documentation?

Not yet

## Checklist:

- [ ] The code has been tested locally
- [ ] Tests have been added to cover all new functionality (unit & integration)
- [ ] The documentation has been updated to reflect any changes
- [ ] The code has been formatted with [pre-commit](https://pre-commit.com/)

### CODE REVIEW

File: brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+def create_atlas(working_dir, resolution):
+    ATLAS_NAME = "axolotl"
```
Comment (alessandrofelder) [inline brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py:25]:
```suggestions
ATLAS_NAME = "unam_axolotl"
```
Our atlas name usually starts with the institution that made the atlas, in this case UNAM :)

File: brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    SPECIES = "Ambystoma mexicanum"
+    ATLAS_LINK = "https://www.nature.com/articles/s41598-021-89357-3#citeas"
+    CITATION = (
+        "Lazcano, I. et al. 2021, https://doi.org/10.1038/s41598-021-89357-3"
+    )
+    ATLAS_FILE_URL = "https://zenodo.org/records/4595016"
+    ORIENTATION = "ras"
+    ROOT_ID_LEFT = 42  # FIXME
+    ROOT_ID_RIGHT = 92  # FIXME
+    ATLAS_PACKAGER = "Name, name@gmail.com"  # FIXME
```
Comment (alessandrofelder) [inline brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py:35]: This should be the three of us (@saimaabdus19, @dpshelio and me) - we can give the generic BrainGlobe email hello@brainglobe.info.

**Review by alessandrofelder — COMMENTED** [review]
Thanks for this @saimaabdus19 - looking good.
A lot of the required elements are there - we can discuss how to "glue" them together in our pair-programming session next week :)

File: brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py (hunk 1 above)
Comment (adamltyson) [reply brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py:25]: FYI it doesn't have to (there's no specific convention), but in this case I think it's a good name!

**Review by alessandrofelder — APPROVED** [review]
Approving and merging this, in the interest of tweaking with smaller PRs to finish off #270 - thanks for your contribution @saimaabdus19 :tada:

**Pull request merged**